### DISCUSSION TITLE
Lab-D matmul.hex

### DISCUSSION THREAD

**Discussion by b08901102** [bol-edu/HLS-SOC-Discussions#181] [General]
![螢幕擷取畫面 2023-12-12 134839](https://github.com/bol-edu/HLS-SOC-Discussions/assets/81764578/7210872f-76f3-4273-8ca1-7555c2d670ce)
想請問Lab-D中的Implementation提到的matmul.hex是指哪一個檔案呢? 還是說要從Lab6調用過來呢?

**Comment by alankuo04** [comment]
同學你好

LabD 用到的 matmul.hex 就是 Lab6 使用到的同一個檔案，所以直接調用過來就可以了
